Core/Object Arts/Dolphin/MVP/Views/Common Controls/NMLVCUSTOMDRAW.cls

```diff
@@ -83,9 +89,7 @@ itemHandle
 reset
 	"Reset the canvas to default settings."
 
-	self font
-		isItalic: false;
-		isStruckThrough: false.
+	self canvas font: self view actualFont.
```

Comment:
Hmm, this doesn't respect the ListView custom draw protocol in the case where the previous column had a custom draw block that changed the font, and the current column does not. It will switch the font back to the default, but it won't return CDRF_NEWFONT to tell the control that the font has been changed. Apparently this doesn't actually matter with the current control implementation (at least on Windows 10, I've not tried it elsewhere), but it might not work on other implementations either now or in the future.